etc/hibinit-config.cfg

```diff
@@ -28,4 +28,4 @@ target-size-mb = 4000
 mkswap = mkswap {swapfile}
 # The command used to turn on the swap
 swapon = swapon {swapfile}
-swapoff = swapoff {swapfile}
+swapoff = swapoff {swapfile}
```

Comment:
What really changed here ? Sorry i didnt understand the icon

Reply:
No change, think this is the whitespace being picked up